### ISSUE TITLE
Regression commit #581995 

### ISSUE THREAD

**Issue by fmorency** [tpoechtrager/osxcross#200]
Hi,
Commit  `581995` is causing a regression here when cross-compiling our Rust toolkit to macOS. Using the `rust:stretch` docker image

```
$ docker run -it --rm -v /tmp:/tmp rust:stretch
```
run
```
$ apt update && apt install -y git build-essential clang cmake zlib1g-dev libmpc-dev libmpfr-dev libgmp-dev
$ git clone https://github.com/tpoechtrager/osxcross
$ cd osxcross
$ wget https://s3.dockerproject.org/darwin/v2/MacOSX10.11.sdk.tar.xz
$ mv MacOSX10.11.sdk.tar.xz tarballs/
$ sed -i -e 's|-march=native||g' build_clang.sh wrapper/build.sh
$ UNATTENDED=yes OSX_VERSION_MIN=10.8 ./build.sh
```

The `build.sh` script exits with

```
extracting MacOSX10.11.sdk.tar.xz ...

## Building wrapper ##


exiting with abnormal exit code (1)
run 'OCDEBUG=1 ./build.sh' to enable debug messages
```

Previous commit `d3392f4` works just fine.

**Comment by tpoechtrager** [comment]
Should be fixed.

**Comment by fmorency** [comment]
Fixed. Thank you!

**Comment by erijo** [comment]
Note that the fix in 0713de1 isn't correct. `||` should be `&&`. Otherwise the link is created when dsymutil doesn't exist.

**Comment by tpoechtrager** [comment]
It's the wanted behavior.

Erik Johansson <notifications@github.com> schrieb am Mi., 23. Okt. 2019,
10:50:
